flipt-engine/src/models/common.rs

```diff
@@ -9,6 +9,15 @@ pub enum FlagType {
     Boolean,
 }
 
+impl From<i32> for FlagType {
+    fn from(n: i32) -> FlagType {
+        match n {
+            1 => FlagType::Boolean,
+            _ => FlagType::Variant,
```

Comment:
```suggestion
            0 => FlagType::Variant,
            _ => FlagType::Variant,
```

Maybe we should be all the way explicit here.